packages/spark-extras/components/dictionary/_dictionary.scss

```diff
@@ -24,22 +25,34 @@ $dictionary-breakpoint: 38.4375rem !default;
   }
 }
 
+.sprk-c-Dictionary .sprk-b-Label {
+  font-size: $font-size-body-four;
```

Comment:
The key is smaller than the value. Is that supposed to be like that?

Reply:
by design

Reply:
lol the only time i've use the thumbs down 😩